CMakeLists.txt

```diff
@@ -109,7 +109,7 @@ set_target_properties(GMP::GMP PROPERTIES
 )
 
 function(add_sail_wrapper _format _suffix _sanitizer_flags)
-    add_library(sail_wrapper_${_format}${_suffix} STATIC test/sail_wrapper_${_format}.c test/contrib/sail.c test/contrib/rts.c test/contrib/sail_failure.c)
+    add_library(sail_wrapper_${_format}${_suffix} STATIC test/sail_wrapper_${_format}.c test/contrib/sail.c test/contrib/sail_failure.c)
```

Comment:
Can we delete the file if we don't need it any more?

Reply:
Sigh, I must have brought it back because I changed the name of the folder while rebasing. Thanks for all the review - hopefully done once I push that change...